### PR TITLE
Dockershim critest on Windows

### PR DESCRIPTION
**Pull request by hickeyma**
Partial #613

Extracted from #611. Refer to https://github.com/kubernetes-sigs/cri-tools/pull/611#discussion_r448425456 for more details.

### CODE REVIEW

**Comment by hickeyma** [comment]
Hey @marosset. You mentioned a while back that you might be able to bring some expertise from the Windows side. Let me know if you can provide some input to the failing tests. Thanks.

**Comment by marosset** [comment]
Hi @hickeyma - yes I can help take a look.
Which tests are failing? it looks like all the tests pass PR checks are passing.

**Comment by hickeyma** [comment]
That would be great @marosset. Windows was temporarily removed from the matrix after the last commit, so just added it back in.

**Comment by hickeyma** [comment]
Sorted out issues and workflow now ready for review.

File: .github/workflows/dockershim.yml [hunk 1]
```diff
@@ -43,31 +42,118 @@ jobs:
...
+          set -o pipefail
+          set -x
+
+          export LANG=C
+          export LC_ALL=C
+          FOCUS="${FOCUS:-"Conformance"}"
+          REPORT_DIR="${REPORT_DIR:-"c/_artifacts"}"
+          mkdir -p "${REPORT_DIR}"
+
+          # Skipping these tests because there are no manifests for windows/amd64
```
Comment (mikebrow) [inline .github/workflows/dockershim.yml:106]: nit.. TODO the skips..

**Review by mikebrow — COMMENTED** [review]
looking good

File: .github/workflows/dockershim.yml (hunk 1 above)
Comment (hickeyma) [reply .github/workflows/dockershim.yml]: Done

**Comment by hickeyma** [comment]
BTW, I slipped some small comments in files in `images/image-user/` directory. This is so can kick off post submit jobs in test-infra. I can push them to another PR if need be.

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**